themis.podspec

```diff
@@ -77,17 +82,33 @@ Pod::Spec.new do |s|
 
     # use `themis/themis-openssl` as separate target to use Themis with OpenSSL
     s.subspec 'themis-openssl' do |so|
-        # Enable bitcode for openssl only, unfortunately boringssl with bitcode not available at the moment
-        so.ios.pod_target_xcconfig = {'ENABLE_BITCODE' => 'YES' }
-
+        # Enable bitcode for OpenSSL in a very specific way, but it works, thanks to @deszip 
+        #so.ios.pod_target_xcconfig = {'ENABLE_BITCODE' => 'YES' }
+        so.ios.pod_target_xcconfig = {
+            'OTHER_CFLAGS[config=Debug]'                => '$(inherited) -fembed-bitcode-marker',
+            'OTHER_CFLAGS[config=Release]'              => '$(inherited) -fembed-bitcode',
+            'BITCODE_GENERATION_MODE[config=Release]'   => 'bitcode',
+            'BITCODE_GENERATION_MODE[config=Debug]'     => 'bitcode-marker'
+        }
+        
+        # Xcode12, arm64 simulator issues https://stackoverflow.com/a/63955114
+        # disable building for arm64 simulator for now
+        
+        so.ios.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
+        so.ios.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

Comment:
Oh, this is a nice way to handle it :+1: Should save the users trouble of disabling it manually. (Which I presume they will have to do with Carthage?)

Reply:
I'm a fan of changing user target options, and prefer the users do that themselves. But looks like it's impossible to lint podspec without adding these, because podspec-linting-test-projects are generated automatically as default projects, noone can disable arm64 in them